erica/application/FreischaltCode/FreischaltCode.py

```diff
@@ -13,6 +13,7 @@
 class FreischaltCodeRequestPayloadDto(BaseDto):
     tax_id_number: TaxIdNumber
     date_of_birth: date
+    tax_year: str = "2021"
```

Comment:
Why not use the constant VERANLAGUNGSZEITRAUM for all these places?

Reply:
Honestly, because Honestly - because I don't fully understand layered Architecture and thought this might be a violation of the one-direction-access rule? But maybe that's Quatsch. I'll use it.